### DISCUSSION TITLE
Fixing Symfony missing type spec messages

### DISCUSSION THREAD

**Discussion by slashrsm** [mpdf/mpdf#1737] [Q&A]
We have a Symfony 6 project where we use `mpdf/mpdf`. When running our test suite we get a bunch of messages like this. 

```
  1x: Method "Psr\Log\LoggerAwareInterface::setLogger()" might add "void" as a native return type declaration in the future. Do the same in implementation "Mpdf\SizeConverter" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in PdfRendererTest::testSavePdf from App\Tests\Pdf
```

The solution is to either add native type specs to the functions that are affected or add a `@return` into the doc block. Which approach would be preferred? I suspect that native type specs won't play since we have to support PHP < 7.4?

**Comment by finwe** [comment]
Exactly, native type specs won't play. See https://github.com/mpdf/mpdf/issues/1622

**Comment by slashrsm** [comment]
@finwe would PHPDoc type specs work?
